components/cluster-secret-store/production/insights-approle-id-patch.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/provider/vault/auth/appRole/roleId
+  value: 573e6024-0c58-7e95-365b-d2ad6127e838
```

Comment:
Is it intentional to have the same app-role in stating and production?

Reply:
I'm not sure if the secret is needed in staging at all, but we don't have a separate staging one. I guess it's safe as we use the secrets only for RHEL subscription and it does not matter where the built containers get the subscription context from.